Assets/AnkrSDK/Scripts/WebGL/Implementation/EthHandlerWebGL.cs

```diff
@@ -88,5 +96,76 @@ public Task<string> Sign(string messageToSign, string address)
 
 			return _webGlWrapper.SendTransaction(transactionData);
 		}
+		
+		public async Task<BigInteger> GetBalance(string address = null)
+		{
+			address = await GetDefaultAccount();
+			var callObject = new WebGLCallObject
+			{
+				Path = _getBalanceMethodName,
+				Args = address != null ? new Object[] {address} : null
```

Comment:
I think you can just make it new []{}

Reply:
or at least make it object[]

Reply:
Object is a system type while object is an alias to it. You should prefer alias at 95% of the times.